README.md

```diff
@@ -17,56 +11,136 @@
 - [License](#license)
 - [Copyright](#copyright)
 
-
 # OpenSearch SQL
 
 OpenSearch enables you to extract insights out of OpenSearch using the familiar SQL or Piped Processing Language (PPL) query syntax. Use aggregations, group by, and where clauses to investigate your data. Read your data as JSON documents or CSV tables so you have the flexibility to use the format that works best for you.
 
 The following projects have been merged into this repository as separate folders as of July 9, 2020. Please refer to links below for details. This document will focus on the SQL plugin for OpenSearch.
 
-* [SQL CLI](https://github.com/opensearch-project/sql/tree/main/sql-cli)
-* [SQL JDBC](https://github.com/opensearch-project/sql/tree/main/sql-jdbc)
-* [SQL ODBC](https://github.com/opensearch-project/sql/tree/main/sql-odbc)
-* [Query Workbench](https://github.com/opensearch-project/sql/tree/main/workbench)
-
+- [SQL CLI](https://github.com/opensearch-project/sql/tree/main/sql-cli)
+- [SQL JDBC](https://github.com/opensearch-project/sql/tree/main/sql-jdbc)
+- [SQL ODBC](https://github.com/opensearch-project/sql/tree/main/sql-odbc)
+- [Query Workbench](https://github.com/opensearch-project/sql/tree/main/workbench)
+
+## Code Summary
+
+### SQL Engine
+
+|                              |                                                                                                                                              |
```

Comment:
are the headers left empty on purpose?

Reply:
yes, that's a dummy header for the table. Without these md won't render it as a table.